measures/ResStockArguments/measure.rb

```diff
@@ -332,6 +337,11 @@ def run(model, runner, user_arguments)
       args['geometry_unit_cfa'] = Float(args['geometry_unit_cfa'])
     end
 
+    # Vintage
+    if args['vintage'].is_initialized
+      args['year_built'] = args['vintage'].get.gsub(/[^0-9]/, '').to_i # strip non-numeric
```

Comment:
I suggest changing this to `Integer(Float(foo))` instead of `.to_i`, as the former will throw an error if provided an inappropriate string (which we'd like to know so we can fix the issue) while the latter will simply return zero. E.g.:

```
> Integer(Float("foo"))
ArgumentError: invalid value for Float(): "foo"
        from (irb):1:in `Float'
        from (irb):1
        from C:/Ruby22-x64/bin/irb:11:in `<main>'
> "foo".to_i
=> 0
```